allennlp/service/server_sanic.py

```diff
@@ -8,34 +8,36 @@ def run(port: int) -> None:
     print("Starting a sanic server on port {}.".format(port))
     app = make_app()
     # TODO(joelgrus): make this configurable
-    app.servables = ServableCollection.default()
+    app.predictors = PredictorCollection.default()
```

Comment:
I see, this is where the `PredictorCollection` is needed.  Seems like this should just be part of configuring and instantiating the app, though, instead of having a bunch of hard-coded predictors from the registry.  Like, the app could have a mapping from names to `Predictors`, where it instantiates each `Predictor` with a config.  That mapping gets constructed when the app starts, and just lives inside the app.  Then you don't need an auxiliary object to encode what things the app can serve.